docs.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = () => ({
+  packages: 'packages/*',
+  docs: 'docs',
+  siteName: 'Brisk Docs Docs',
+})
```

Comment:
ESLint will probably complain about this once we've got CI up and running

Reply:
bloody hell. My editor isnt showing me these errors